incorrect return value

>srt\srtcore\core.cpp(9707): warning C4305: 'return': truncation from 'int' to 'bool'

https://github.com/Haivision/srt/blob/49fb18e2b039242e46b306f23b9886be692f074c/srtcore/core.cpp#L9707

--- a/srtcore/buffer_rcv.h
+++ b/srtcore/buffer_rcv.h
@@ -226,7 +226,7 @@ public: // Used for testing
 private:
     inline int incPos(int pos, int inc = 1) const { return (pos + inc) % m_szSize; }
     inline int decPos(int pos) const { return (pos - 1) >= 0 ? (pos - 1) : int(m_szSize - 1); }
-    inline int offPos(int pos1, int pos2) const { return (pos2 >= pos1) ? (pos2 - pos1) : (m_szSize + pos2 - pos1); }
+    inline int offPos(int pos1, int pos2) const { return (pos2 >= pos1) ? (pos2 - pos1) : int(m_szSize + pos2 - pos1); }
 
 private:
     void countBytes(int pkts, int bytes);

--- a/srtcore/core.cpp
+++ b/srtcore/core.cpp
@@ -9704,7 +9704,7 @@ bool srt::CUDT::packUniqueData(CPacket& w_packet, time_point& w_origintime)
             // Encryption failed
             //>>Add stats for crypto failure
             LOGC(qslog.Warn, log << "ENCRYPT FAILED - packet won't be sent, size=" << pld_size);
-            return -1;
+            return false;
         }
     }
 

